Harvest.Web/Startup.cs

```diff
@@ -55,8 +57,11 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
         {
+            LogConfiguration.Setup(Configuration);
+            loggerFactory.AddSerilog();

```

Comment:
I didn't think this would still be a thing anymore -- I didn't recall seeing it in the more modern examples. what does it do?

Reply:
Copied from Peaks. Apparently nothing if you don't provide a customized Serilog ILogger argument. I'll remove it.